code/modules/status_system/statusEffects.dm

```diff
@@ -3122,9 +3125,13 @@
 				src.final_msg_given = TRUE
 				H.playsound_local(H, 'sound/ambience/spooky/Void_Calls.ogg', 75, FALSE)
 			if (H.bioHolder.age >= src.original_age + 120)
+				H.visible_message(SPAN_ALERT("[H] collapses into a pile of bones!"))
 				H.death(FALSE)
-				H.skeletonize()
-				src.linked_curser.lift_curse_specific(FALSE, H)
+				H.set_mutantrace(/datum/mutantrace/skeleton)
```

Comment:
Does adding the mutantrace after death still make them fall apart or do they get stuck in a weird dead-but-intact skeleton state?

Reply:
switched order so they fall apart properly